src/main/java/lotto/ui/ResultView.java

```diff
@@ -1,14 +1,69 @@
 package lotto.ui;
 
+import lotto.domain.LottoPrize;
+import lotto.domain.LottoStatistics;
+import lotto.domain.LottoTicket;
+import lotto.domain.LottoTickets;
+
 import java.text.MessageFormat;
+import java.util.List;
 
 public class ResultView {
 
-    public static void printExpressionResult(int result) {
-        System.out.println(MessageFormat.format("결과: {0}", result));
+    public static void printLottoTickets(LottoTickets lottoTickets) {
+        String message = formatLottoTicketCount(lottoTickets) + formatLottoTicketNumbers(lottoTickets);
+        System.out.println(message);
+    }
+
+    private static StringBuilder formatLottoTicketCount(LottoTickets lottoTickets) {
+        StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.append(formatPurchaseCount(lottoTickets.size()));
+        stringBuilder.append(System.lineSeparator());
+        return stringBuilder;
+    }
+
+    private static String formatLottoTicketNumbers(LottoTickets lottoTickets) {
+        StringBuilder stringBuilder = new StringBuilder();
+        for (LottoTicket ticket : lottoTickets.get()) {
+            stringBuilder.append(ticket.get()).append(System.lineSeparator());
+        }
+        stringBuilder.append(System.lineSeparator());
+        return stringBuilder.toString();
+    }
+
+    private static String formatPurchaseCount(int count) {
+        return MessageFormat.format("{0}개를 구매했습니다.", count);
+    }
+
+    public static void printLottoStatistics(LottoStatistics statisticsMap, int purchaseAmount) {
+        String message = formatLottoPrizes(statisticsMap) + formatProfitRate(statisticsMap, purchaseAmount);
+        System.out.println(message);
+    }
+
+    private static String formatLottoPrizes(LottoStatistics statisticsMap) {
+        List<LottoPrize> prizes = LottoPrize.getLottoPrizes();
+
+        StringBuilder stringBuilder = new StringBuilder();
+        for (LottoPrize prize : prizes) {
+            stringBuilder.append(formatStatistic(statisticsMap, prize)).append(System.lineSeparator());
+        }
+        return stringBuilder.toString();
+    }
+
+    private static String formatStatistic(LottoStatistics statisticsMap, LottoPrize prize) {
+        return MessageFormat.format("{0}개 일치 ({1}원) - {2}개"
+                , LottoPrize.getMatchCount(prize), LottoPrize.getPrize(prize), statisticsMap.getMatchCount(prize));
```

Comment:
```suggestion
                , prize.matchCount(), prize.getPrize(), statisticsMap.getMatchCount(prize));
```
위에서 드렸던 피드백을 반영한다면 
이런 표현이 되겠네요 😄